src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalCatchCheckTest.java

```diff
@@ -87,11 +87,21 @@ public void testIllegalClassNamesBad() throws Exception {
     @Test
     public void testMultipleTypes() throws Exception {
         final DefaultConfiguration checkConfig = createModuleConfig(IllegalCatchCheck.class);
+        checkConfig.addAttribute("illegalClassNames",
```

Comment:
please use `addProperty`, we just merged update for this class, and Configuration.xxxAttributeXxxx are deprecated.

in all places

Reply:
done, separate commit at https://github.com/checkstyle/checkstyle/pull/10136/commits/eb7af217b27a6dade42cd8eb955335f0cdd5cd7c to update usage from https://github.com/checkstyle/checkstyle/pull/10122